ci-operator/step-registry/ovn/sdn-migration-rollback/ovn-sdn-migration-rollback-commands.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -x
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TARGET=${TARGET:-OpenShiftSDN}
+
+oc patch MachineConfigPool master --type='merge' --patch '{"spec":{"paused":true}}'
+oc patch MachineConfigPool worker --type='merge' --patch '{"spec":{"paused":true}}'
+
+# Reset the spec.migration before we can set it to other value
+oc patch Network.operator.openshift.io cluster --type='merge' --patch '{"spec":{"migration":null}}'
+# Wait until CNO update the applied-cluster cm to the latest
+sleep 10
+
+# Allow rollback
+# Change network to target network in Network.config.openshift.io the CR to trigger machine config update by MCO.
+oc patch Network.operator.openshift.io cluster --type='merge' --patch "{\"spec\":{\"migration\":{\"networkType\":\"${TARGET}\"}}}"
+oc patch Network.config.openshift.io cluster --type='merge' --patch "{\"spec\":{\"networkType\":\"${TARGET}\"}}"
+
+sleep 30
+# Wait until the multus pods are restarted
+timeout 300 oc rollout status ds/multus -n openshift-multus
+
+# Reboot all the nodes
+
+oc get pod -n openshift-machine-config-operator | grep daemon|awk '{print $1}'|xargs -i oc rsh -n openshift-machine-config-operator {} chroot /rootfs shutdown -r +1
+sleep 60
+
+# Resume MCPs after reboot
+timeout 1800s bash <<EOT
+until
+  oc patch MachineConfigPool master --type='merge' --patch '{"spec":{"paused":false}}' && \
+  oc patch MachineConfigPool worker --type='merge' --patch '{"spec":{"paused":false}}';
+do 
+  sleep 10;
+done
+EOT
+
+sleep 180
```

Comment:
why do we need this?

Reply:
what is something we can check to know when it would be safe to run that 'oc patch' command? is there some match in the 'oc get co" command we can look for to know when it's safe?